PostHog/PostHogSDK.swift

```diff
@@ -610,6 +618,42 @@ let maxRetryDelay = 30.0
         ))
     }
 
+    func autocapture(
+        eventType: String,
+        elements: [[String: Any]],
+        elementsChain: String,
+        properties: [String: Any]
```

Comment:
I think this one should be nullable? probably not mandatory

Reply:
`autocapture` is public in js-lite at least.

Reply:
it does not need to be public, so your call here, I just checked the js-lite that's why I commented.